cosinnus/views/group.py

```diff
@@ -704,7 +705,7 @@ def get(self, request, *args, **kwars):
         response = HttpResponse(content_type='text/csv')
         response['Content-Disposition'] = 'attachment; filename="{}"'.format(filename)
 
-        header = [_('Workshop username'), _('email'), _('workshops count'), _('has logged in'), _('last login date')]
+        header = [_('Workshop username'), _('email'), _('workshops count'), _('has logged in'), _('last login date'), _('Terms of use accepted')]
```

Comment:
Would suggest removing the translations here to reduce the number of translations, as it is a very techy document only and not strictly needed.
```suggestion
        header = ['Workshop username', 'email', 'workshops count', 'has logged in', 'last login date', 'Terms of service accepted']
```